confignode/pom.xml

```diff
@@ -50,11 +50,6 @@
             <artifactId>iotdb-server</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.iotdb</groupId>
-            <artifactId>iotdb-procedure</artifactId>
-            <version>${project.version}</version>
-        </dependency>
```

Comment:
Why remove procedure here?

Reply:
The procedure module is removed in [this commit](https://github.com/apache/iotdb/commit/fa06bbb43243fef53fa07b82dc848cd3684e09b4). IDE reported an error message about this part.